disciplines/isq.ttl

```diff
@@ -256,6 +477,39 @@ email: emanuele.ghedini@unibo.it"""@en ,
                                            skos:prefLabel "Heat"@en .
 
 
+###  http://emmo.info/emmo#EMMO_12f0b2eb_a7b2_4f87_8176_12748f415832
+:EMMO_12f0b2eb_a7b2_4f87_8176_12748f415832 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_af794e9d_dc7d_4756_83e1_2cd0e2ec864e ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_3227b821_26a5_4c7c_9c01_5c24483e0bd0
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "6-15" ;
+                                           skos:prefLabel "RelativePermittivity"@en .
+
+
+###  http://emmo.info/emmo#EMMO_134b78a0_63f8_4f7f_8098_925d1d934e48
+:EMMO_134b78a0_63f8_4f7f_8098_925d1d934e48 rdf:type owl:Class ;
+                                           rdfs:subClassOf :EMMO_c29603f5_95e8_42f5_ab0c_f3bcf3166d53 ,
+                                                           [ rdf:type owl:Restriction ;
+                                                             owl:onProperty :EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                             owl:someValuesFrom :EMMO_608a1b30_df6f_4bbb_9dc3_5c0de92fd9cf
+                                                           ] ;
+                                           :EMMO_8de5d5bf_db1c_40ac_b698_095ba3b18578 "12-2.2" ;
+                                           :EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "fundamental translation vectors for the reciprocal lattice"@en ;
+                                           skos:prefLabel "FundamentalReciprocalLatticeVectors"@en .
```

Comment:
Suggest to change to singular: FundamentalReciprocalLatticeVector

Reply:
done